types/multi-progress/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
Why are strictNullChecks disabled?

Reply:
I just copied these in from another folder in the project (progress), I've turned it back on.